tests/serializers/models/multi_table.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+
+class SuperClassManager(models.Manager):
+    def get_by_natural_key(self, code):
+        return self.get(code=code)
+
+
+class SuperClass(models.Model):
```

Comment:
I think `Parent` and `Child` model names would be more in line with other model names I've seen.